### ISSUE TITLE
Feature request: Pagination

### ISSUE THREAD

**Issue by kunaakos** [nolenroyalty/every-uuid#7]
## The Issue

**This is absolutely not about making google index every UUID yet to be calculated.**

Because as a human being with a legitimate interest in UUIDs, I find it cumbersome to always scroll to my favorite one: `9855abc5-ea49-453e-81ff-3099a14f8a22`. I have to look up my notes and copy it from there into the search bar to find it every time. Using the favorites is not an option for me, because I use a large variety of devices to watch UUIDs, and favorites are currently saved in local storage.

## Proposed solution

Rendering UUIDs in a paginated fashion would allow me to save a link to the page that contains my favorite UUID to my bookmarks - which are then synced through the cloud. This could also solve #5 - especially if combined with deep linking.

## Implementation notes

This new experience should be released on a different page, e.g. `everyuuid.com/bythedozen`, where only 12 UUIDs are displayed per page, with an added [pagination component](https://design-system.w3.org/components/pagination.html). This preserves the current experience on the existing URL, so people coming from social media are greeted with the same version they saw in the video(s).

Each page should be accessible on, say `everyuuid.com/bythedozen?page=n` where `n` is 1-indexed and `page` is an optional parameter that defaults to 0.

## TBD
I will not bore ya with the SEO and rendering issues unless interested, but this is certainly doable (I read through the code). Worst case scenario it requires switching to hybrid SSR, which - given the amount of pages and your already high traffic - might exhaust quotas or run up bills.

I am here to serve the community of UUID enthusiasts by opening a PR if you decide to go ahead with this.

**Comment by Pdzly** [comment]
Why not making rather make a export favorites option?

**Comment by To1ne** [comment]
> bookmark option like .../{insertuuidhere} that you can bookmark / share <= that already works.

Yes `https://everyuuid.com/#9855abc5-ea49-453e-81ff-3099a14f8a22` would be exactly what we need.

**Comment by kunaakos** [comment]
I was being tongue-in-cheek in my description. The point would to structure the pages in a way that would make bots crawl them, because they'd see every individual UUID as a separate resource which links to other ones. Similar techniques are used to trap AI crawlers, see https://zadzmo.org/code/nepenthes/
